src/scout_apm/django/middleware.py

```diff
@@ -22,38 +25,78 @@
     from django.core.urlresolvers import get_urlconf
 
 
-def get_operation_name(request):
+def get_controller_name(request):
     view_func = request.resolver_match.func
     view_name = request.resolver_match._func_path
 
-    if hasattr(view_func, "model_admin"):
-        # Seems to comes from Django admin (attribute only set on Django 1.9+)
-        admin_class = view_func.model_admin.__class__
-        view_name = (
-            admin_class.__module__
-            + "."
-            + admin_class.__name__
-            + "."
-            + view_func.__name__
+    django_admin_components = _get_django_admin_components(view_func)
+    if django_admin_components:
+        view_name = "{}.{}.{}".format(
+            django_admin_components.module,
+            django_admin_components.controller,
+            django_admin_components.action,
+        )
+
+    django_rest_framework_components = _get_django_rest_framework_components(request)
+    if django_rest_framework_components is not None:
+        view_name = "{}.{}".format(
```

Comment:
One thing to investigate is whether we can and do we want to change the operation name here to include the `module`.